tests/broadcast/broadcast.test.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-undef */
+import { html, fixture, expect } from '@open-wc/testing';
```

Comment:
I would like to replace the mocha/expect.js testing setup here with tap but everything is run inside a browser context which means no node only modules are allowed. Would like to further investigate the setup generally as its possible to flip a switch and have it run the tests via puppeteer or playwright as well.